src/transform/transform.ts

```diff
@@ -770,6 +778,10 @@ function generateLROPollerName(schemaResp: SchemaResponse): string {
 
 function generateLROResponseType(response: Response, op: Operation, codeModel: CodeModel) {
   const respTypeName = generateLROResponseTypeName(response);
+  if (isPageableOperation(op)) {
```

Comment:
This block should probably go into `generateLROResponseTypeName()` to keep all the naming in one place.

Reply:
Done